client/src/components/new-request/search.vue

```diff
@@ -119,17 +190,26 @@ export default class NewSearch extends Vue {
       let { value } = newReqModule.entityTypesXPROData.find(ent => ent.rank === 1)
       newReqModule.mutateEntityType(value)
     }
-    if (['CNV', 'MVE'].includes(newVal)) {
+    if (['CNV'].includes(newVal)) {
       this.location = 'BC'
       return
     }
-    if (['ASSUMED', 'MVE'].includes(newVal)) {
+    if (['ASSUMED'].includes(newVal)) {
       if (this.location === 'BC') {
         this.location = 'CA'
       }
     }
   }
-
+  private request_action_enum = [
+    'NEW',
+    'MVE',
+    'REH',
+    'AML',
+    'CHG'
+  ]
+  private get isScreenLg () {
+    return this.$vuetify.breakpoint.width > 1440
```

Comment:
Why not use CSS breakpoints?

https://www.w3schools.com/css/css_rwd_mediaqueries.asp and others.

Reply:
Or Vuetify breakpoints?

https://vuetifyjs.com/en/features/breakpoints/

Reply:
Yea, the idea WAS to use Vuetify breakpoints here but i complicated it by comparing it to a hard value.
Should be more like `return this.$vuetify.breakpoint.lg`.

Reply:
> `return this.$vuetify.breakpoint.lg`

I just learned something. That would be perfect here. Cool!

Reply:
Me as well! Interesting that it is a boolean eh

Reply:
🤓